integration-tests/src/test/kotlin/org/stellar/anchor/platform/AnchorPlatformIntegrationTest.kt

```diff
@@ -325,7 +331,7 @@ class AnchorPlatformIntegrationTest {
         "sep38.quoteIntegrationEndPoint" to "http://localhost:8081",
         "payment-gateway.circle.name" to "circle",
         "payment-gateway.circle.enabled" to "true",
-        "spring.jpa.properties.hibernate.dialect" to "org.hibernate.dialect.H2Dialect",
+        "spring.jpa.database-platform" to "org.stellar.anchor.platform.sqlite.SQLiteDialect",
```

Comment:
Can we delay the change to post config refactoring?

Reply:
The thing is H2 won't work anymore because it cannot be shared by 2 separate services.

Reply:
Oh, that's right. H2 in-memory process cannot be shared but H2 in server mode can be shared. 

Based on this [cheatsheet](http://www.h2database.com/html/cheatSheet.html), running a separate H2 may be more complicated than sharing the file based SQLite. I think it makes more sense to use SQLite.